### PR TITLE
Added spanish

### PR DESCRIPTION
**Pull request by lozalojo**

### CODE REVIEW

File: module.json [hunk 1]
```diff
@@ -44,7 +44,12 @@
 			"lang": "zh-tw",
 			"name": "正體中文",
 			"path": "lang/zh-tw.json"
-		}
+		},
+        {
```
Comment (manuelVo) [inline module.json:48]: Please use tabs for those indentation as is done in the rest of the file.

**Review by manuelVo — CHANGES_REQUESTED** [review]

File: module.json (hunk 1 above)
Comment (lozalojo) [reply module.json]: Fixed, I usually pay attention to indentation, but I missed two lines this time, sorry.

**Review by manuelVo — APPROVED** [review]

**Comment by manuelVo** [comment]
No worries, and thanks for the translation.

Maybe you want to look into installing an extension for "editorconfig" support. Editorconfig is a way to supply configuration settings like indentation style, newline style and such to the editor on a per-project basis and if a project contains such a configuration file that config will be used instead of the default configuration that's put into your editors. Many projects supply these `.editorconfig` files, which then prevents such a thing from happening.

**Comment by manuelVo** [comment]
With what name would you like to be attributed for the translation in the readme and changelog. Your github name, your real name or some other name?

**Comment by lozalojo** [comment]
@Viriato139ac#0342

**Pull request merged**